mariamihaelafainarea/assignment1/SinglyLinkedList

```diff
@@ -0,0 +1,183 @@
+import java.util.Scanner;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author maria
+ */
+//i cannot get rid of request changes thing. 
```

Comment:
Maybe try and remove those lines now, I will just make this review iteration as "comment" and not "request changes" :)

Reply:
i have deleted it